### ISSUE TITLE
Feature Request: Add GitLabs GraphQL API

### ISSUE THREAD

**Issue by fridlmue** [black-sheep-dev/harbour-sailhub#44]
Very useful App. Thanks a lot!
I think GitLab has a GraphQL-API as well. 
[GtiLab GraphQL Doku](https://docs.gitlab.com/ee/api/graphql/)
[GtiLab GraphQL Doku - Getting Started](https://docs.gitlab.com/ee/api/graphql/getting_started.html)

I don't know how much it differs from the GItHub-API, but it would be awesome for sure to have both the major Git-Platforms in this App.

**Comment by black-sheep-dev** [comment]
Thanks for your feedback!

I checked the GitLab GraphQL Documentation. It is similar to the GitHub API but some data is handled in a different way.
It seems not impossible to integrate this, only some other queries and parsing things.
At the moment I try to realize the same functionality like the official Android app and I don't know at the moment  if the GitLab API will fit into this "UI"?!?

My plan is to finish the basic app functionality for GitHub first to get a full overview what is possible. Afterwards I will check if both APIs can be combined with a single UI.

**Comment by fridlmue** [comment]
Yeah, sure! This was just a Idea that came up while using the App! Thank you for possibly considering it in the future.
